utilities/pulumi_state_splitter/tests/test_split.py

```diff
@@ -197,6 +204,7 @@ def test_load(self):
         for state in state_dir.state, want:
             state.checkpoint.latest.resources.sort(key=util.resource_key)
             for resource in state.checkpoint.latest.resources:
+                _maybe_delattr(resource, "sourcePosition")
```

Comment:
**Optional nit:** "maybe delete" sounds a bit vague (under what conditions it will be removed)? Maybe `_delattr_if_exists` or `_safe_delattr`?

Reply:
Done in 2d3118369d42876dd02019335b50373c0246299f .